checker/manual/creating-a-checker.tex

```diff
@@ -516,23 +499,19 @@ \subsection{Declaratively specifying implicit annotations\label{declarative-type
 %END LATEX
 \begin{Verbatim}
   @SubtypeOf( { Nullable.class } )
-  @ImplicitFor(
-    types={TypeKind.PACKAGE},
-    typeClasses={AnnotatedPrimitiveType.class},
-    trees={
-      Tree.Kind.NEW_CLASS,
-      Tree.Kind.NEW_ARRAY,
-      Tree.Kind.PLUS,
+  @ImplicitFor(types = { TypeKind.PACKAGE, TypeKind.INT,
```

Comment:
The primitive types TypeKind.BYTE and TypeKind.SHORT are missing from this list, and I'm not sure why.  (Is it because there aren't literals of those types, and this is only intended for literals?  A comment to that effect would be helpful, though given the "literals =" below I don't think that is the case.